libs/api/protocol/src/lib/infra/service/unit-input.transformer.spec.ts

```diff
@@ -33,65 +33,17 @@ const successfulCases: {
 	},
 ];
 
-const unsuccessfulCases: { case: string; input: UnitInput }[] = [
```

Comment:
With removing all the unsuccessful cases, you leave the transformer's `default` statement untested.

Reply:
added a test for unknown types